src/CMakeLists.txt

```diff
@@ -41,10 +44,6 @@ target_compile_definitions(dart_dll PRIVATE
 )
 
 if(WIN32)
-    set_property(TARGET dart_dll PROPERTY
```

Comment:
Why remove this?

Reply:
set_property(TARGET dart_dll PROPERTY
             MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
   The Line was Double

Reply:
The Line is double